File: C/categorize.c
#include<stdio.h>

int main(){
    float h;
    scanf("%f",&h);

    if(h<150)
        printf("Person is Dwarf.");
    else if(h<165)
        printf("Person is average heighted.");
    else if(h<195)
        printf("Person is taller.");
    else
        printf("Abnormal height.");

    return 0;
}

File: java/categorize.java
import java.util.Scanner;

public class CF{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        float h = scanner.nextFloat();
        if (h < 150) {
            System.out.println("Person is Dwarf.");
        }
        else if (h <= 165) {
            System.out.println("Person is average heighted.");
        }
        else if (h <= 195) {
            System.out.println("Person is taller.");
        }
        else{
            System.out.println("Abnormal height.");
        }
    }
}
